README.adoc

```diff
@@ -15,18 +22,543 @@ image:https://circleci.com/gh/edn-query-language/eql/tree/master.svg?style=svg["
 This repository contains a specification for EQL and a support library to help the development
 of tools using it.
 
-== Rational
+== What is EQL?
 
-*EQL* is a query syntax to express information requirements using link:https://github.com/edn-format/edn[edn].
+EQL is a declarative way to make hierarchical (and possibly nested) selections of information about data requirements.
+
+EQL doesn't have its own syntax; it uses EDN to express the request, taking advantage of
```

Comment:
Wilker, I don't think this is a true statement.  "Syntax" means "the arrangement of words and phrases to create well-formed sentences in a language", and EQL definitely has requirements for how you arrange the things in the notation.  EDN itself is the base-level syntax, but the additional requirements EQL puts on top are technically EQL's syntax, wouldn't you say?

Reply:
Yes, I think you are right, syntax was a bad term, what I'm trying to say is that it doesn't have its own language, so I'll change for that, what do you think?